.github/workflows/liquidation.yml

```
@@ -0,0 +1,135 @@
+name: Liquidation Indexing
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Start A3P container
+        run: docker compose --profile ci up -d a3p
+
+      - run: corepack enable
+        shell: bash
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Start subql indexer
+        env:
+          AGORIC_NET: ci
+        run: yarn dev
+
+      - name: Print initial logs of all containers
+        run: |
+          echo "Fetching initial logs for all containers..."
+          containers=$(docker ps --format '{{.ID}}')
+          for container in $containers; do
+            echo "Fetching initial logs for container $container..."
+            docker logs $container
+          done
+
+      - name: Set ATOM Price to 12.34
+        run: node .github/scripts/changePrice.mjs
+        env:
+          amount: 12.34
+          containerName: agd
+          agoricNet: local
+
+      - name: Get active vaults
+        run: node .github/scripts/getActiveVaults.mjs
+        env:
+          apiUrl: 'http://localhost:3000/'
+          expectedVaults: 7
+
+      - name: Create Vault with 100 Minted and 15 Collateral
+        run: node .github/scripts/createVault.mjs
+        env:
+          wantMinted: '100'
+          giveCollateral: '15'
+          userKey: 'gov3'
+          agoricNet: 'local'
+          commandTimeout: '120'
+          containerName: 'agd'
+
+      - name: Create Vault with 103 Minted and 15 Collateral
+        run: node .github/scripts/createVault.mjs
+        env:
+          wantMinted: '103'
+          giveCollateral: '15'
+          userKey: 'gov3'
+          agoricNet: 'local'
+          commandTimeout: '120'
+          containerName: 'agd'
+
+      - name: Create Vault with 105 Minted and 15 Collateral
+        run: node .github/scripts/createVault.mjs
+        env:
+          wantMinted: '105'
+          giveCollateral: '15'
+          userKey: 'gov3'
+          agoricNet: 'local'
+          commandTimeout: '120'
+          containerName: 'agd'
+
+      - name: Get active vaults
+        run: node .github/scripts/getActiveVaults.mjs
+        env:
+          apiUrl: 'http://localhost:3000/'
+          expectedVaults: 10
+
+      - name: Place bid for 90IST
+        run: node .github/scripts/placeBid.mjs
```

Once you add the shebang you don't have to specify the interpreter at the callsite. It shouldn't be the concern of the caller how to run it.
```suggestion
        run: .github/scripts/placeBid.mjs
```

Separately, these scripts have nothing to do with Github. Please put them somewhere else in the repo. e.g. `/scripts`.
```suggestion
        run: ./scripts/placeBid.mjs
```